config/hazelcast-client.yaml

```diff
@@ -15,6 +15,10 @@
 # limitations under the License.
 #
 
-seatunnel:
-    engine:
-        backup-count: 1
\ No newline at end of file
+hazelcast-client:
+  cluster-name: seatunnel
+  properties:
+      hazelcast.logging.type: log4j2
+  network:
+    cluster-members:
+      - localhost:5801
```

Comment:
We also need detailed documentation for various parameters

Reply:
yes.